### ISSUE TITLE
MSYS - locale not valid

### ISSUE THREAD

**Issue by tomangelo2** [colobot/colobot#1735]
When trying to run a binary compiled with MSYS/MinGW this error shows up in console:
```
terminate called after throwing an instance of 'std::runtime_error'
  what():  locale::facet::_S_create_c_locale name not valid
```

There is no backtrace, the crash happens even before main() function starts up.

**Comment by hexagonrecursion** [comment]
* I think this might be the same as  #1710
* @tomaszkax86 appears to already be working to fix this #1714

**Comment by tomangelo2** [comment]
Yup, just compiled dev-utf8 branch, it runs fine.

**Comment by hexagonrecursion** [comment]
Fixed by  #1714
